### ISSUE TITLE
[Bug] Cloak of Shadows (BB #796)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#796]
This issue was migrated from bitbucket.
**Original Reporter:** Flywithme
**Original Date:** 27.01.2015 23:25:54 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796

<hr>

http://www.wowwiki.com/Cloak_of_Shadows - Will not provide the rogue immunity to a Hunter's Flare ability.

As we can see on the screenshot - stealth + CoS:
![WoWScrnShot_012715_210512.jpg](https://bitbucket.org/repo/kMeMB/images/3165678996-WoWScrnShot_012715_210512.jpg)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 28.01.2015 00:06:05 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15289826

<hr>

Sorry - my mistake in the proof.

"Will not" - added in September 4, 2010‎ (http://www.wowwiki.com/Cloak_of_Shadows?action=edit&undoafter=2315874&undo=2374425)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 28.01.2015 00:06:46 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15289833

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 29.01.2015 18:20:04 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15336596

<hr>

There are new proof of 100%
http://www.wowwiki.com/Cloak_of_Shadows?direction=prev&oldid=1218512 - 
"_Does not provide the rogue immunity to a Hunter's flare ability._"

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 29.01.2015 18:20:24 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15336604

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 29.01.2015 21:37:53 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15340005

<hr>

Fix:

```
#!c++

DELETE FROM `spell_linked_spell` WHERE  `spell_trigger`=31224 AND `spell_effect`=-1543 AND `type`=2;
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 30.01.2015 15:49:06 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15356908

<hr>

What's the problem here?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 31.01.2015 10:53:41 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15368514

<hr>

Cloak of Shadow should not provide immunity to Flare

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 31.01.2015 12:06:43 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15369277

<hr>

on your screenshot Cloak of Shadow not provides immunity - where is the problem?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 31.01.2015 12:17:07 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15369426

<hr>

If there was no immunity - he lost to stealth.

when the rouge using Cloak of Shadows - no debuff Flare. But should be.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 31.01.2015 12:45:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15369728

<hr>

eeeemmm what?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 31.01.2015 22:06:40 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15376092

<hr>

http://www.wowwiki.com/Cloak_of_Shadows?direction=prev&oldid=1218512 - read and test!!( "Does not provide the rogue immunity to a Hunter's flare ability.")

https://www.youtube.com/watch?v=WB2um7UB548

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 01.02.2015 00:18:26 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15383679

<hr>

Where is IMMUNITY on YOUR SCREENSHOT?

I see that hunter can see ROGUE IN STEALTH due to hunter's flare

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 01.02.2015 00:58:39 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15383898

<hr>

**Rogues can use CoS to ignore flare - This is a bug!**

"Does not provide the rogue immunity to a Hunter's flare ability."

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 01.02.2015 09:30:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15386436

<hr>

@PadreWoW I think he means that there's no "Flare" buff/debuff on the rogue with cloak of shadows

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 08.02.2015 11:19:07 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-15548832

<hr>

This fix - allows you to detect the player if the player use Cloak of Shadows.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 08.04.2015 19:46:36 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/796/#comment-17252935

<hr>

Fix #796

→ <<cset a74bf48879b5>>
